lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java

```diff
@@ -0,0 +1,53 @@
+package acceptance.task;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import acceptance.AbstractAccTest;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import pro.taskana.common.api.BaseQuery.SortDirection;
+import pro.taskana.common.api.KeyDomain;
+import pro.taskana.common.test.security.JaasExtension;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.task.api.TaskQueryColumnName;
+import pro.taskana.task.api.TaskService;
+
+/**
+ * Acceptance test for the usecase of adding/removing an attachment of a task and update the result
+ * correctly.
+ */
+@ExtendWith(JaasExtension.class)
+class QueryTaskListValuesAccTest extends AbstractAccTest {
+
+  @WithAccessId(user = "teamlead-1")
+  @Test
+  void should_ReturnPorTypes_When_QueryingForListOfPorTypesForWorkbasket() {
+    TaskService taskService = taskanaEngine.getTaskService();
+    KeyDomain keyDomain = new KeyDomain("GPK_KSC", "DOMAIN_A");
+    List<String> tasks =
```

Comment:
It's not really tasks that get returned here, right? Should we maybe change this to something like taskPorTypes?